.gitignore

```diff
@@ -1,3 +1,11 @@
 build
 out
 .idea
+HELP.md
+.gradle
```

Comment:
to force Git to ignore already committed files, you have to _remove_ (and commit) those files first :(